README.md

```diff
@@ -157,7 +157,35 @@ if you are planning to maintain a long-term fork that users can update from.
 In case you only care about updates through wiping the data,
 you shouldn't need a binary cache for that.
 
+## Nix flakes
 
+Note that nix flake support is experimental at the moment.
+Still, there's some minimal usage example.
+You can build an activation package by procuring flake-powered nix
+(`nix run nixpkgs.nixFlakes`,
+ `echo "experimental-features = nix-command flakes" >> ~/.config/nix/nix.conf)
+writing a `flake.nix`:
+
+```nix
+{
+  description = "nix-on-droid configuration";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/release-20.09";
+    nix-on-droid.url = "github:t184256/nix-on-droid/master";
+    nix-on-droid.inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  outputs = { nix-on-droid, ... }: {
+    nix-on-droid = (nix-on-droid.lib.aarch64-linux.nix-on-droid {
+      config = ./.config/nixpkgs/nix-on-droid.nix;
+    }).activationPackage;
+  };
+}
+```
+
+building it with `nix build .#nix-on-droid --impure`
+and activating it with `result/activate`.
```

Comment:
I am not sure if running this activate script off the result dir will work nicely with the nix-env --set, we are using to manage our generstions. But because of the experimental state, I can live with that line.